src/documents/ContextAwareness.md

```diff
@@ -0,0 +1,327 @@
+---
+title: ContextAwareness API
```

Comment:
What is the relationship of this API to the Progressive Enhancement stuff? They seem similar. Is this effectively a replacement for that?